core/services/ocr2/plugins/mercury/integration_test.go

```diff
@@ -639,6 +649,7 @@ observationSource = """
 [pluginConfig]
 serverURL = "%[8]s"
 serverPubKey = "%[9]x"
+initialValidFromBlockNumber = %[15]d
```

Comment:
Does the `initial` prefix add necessary context? Or would it still make sense as:
```suggestion
validFromBlockNumber = %[15]d
```

Reply:
It actually does add context, because it's only used once in the beginning for a brand new feed

Reply:
Is it documented somewhere?
Is it meaningfully different from something like an `effectiveBlockNumber`? as in an [Effective Date](https://en.wikipedia.org/wiki/Effective_date#:~:text=An%20effective%20date%20or%20as,event%20occurs%20or%20is%20recorded.)?